frontend/cypress/integration/login/login.spec.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+
+context('Login test', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:4000/login');
+  });
+
+  it('should show register button', () => {
+    cy.get('#btn-register');
+  });
+
+  it('should show and type email', () => {
+    cy.get('#email')
+      .type('admin@gmail.com');
+  });
```

Comment:
I think you can add an statement checking if the input has the value typed, like this:

```suggestion
  it('should show and type email', () => {
    cy.get('#email')
      .type('admin@gmail.com')
      .should('have.value', 'admin@gmail.com');
  });
```

Same thing for the other cases below